docs/src/test-cli-js.md

```diff
@@ -3,83 +3,91 @@ id: test-cli
 title: "Command line"
 ---
 
-## Introduction
+Playwright provides a powerful command line interface for running tests, generating code, debugging, and more. The most up to date list of commands and arguments available on the CLI can always be retrieved via `npx playwright --help`.
 
-Here are the most common options available in the command line.
+## Essential Commands
```

Comment:
I personally don't see any use of these level-2 headings, and would prefer just a plain list of commands. Not blocking on this though.